.github/workflows/lint.yml

```diff
@@ -5,6 +5,14 @@ on:
       - main
   pull_request:
 jobs:
+  prettier:
```

Comment:
If I'm not mistaken, this job is not necessary since the prettier plugin is included in eslint.